pkg/cmd/experimental/syncgroups/groupsyncer.go

```diff
@@ -88,10 +80,20 @@ func (s *LDAPGroupSyncer) GetResultingGroups() ([]*userapi.Group, []error) {
 		// update the OpenShift Group corresponding to this record
 		openshiftGroup, err := s.makeOpenShiftGroup(ldapGroupUID, usernames)
 		if err != nil {
+			fmt.Fprintf(s.Err, "Error building OpenShift group for LDAP group %q: %v.  Continuing with the next LDAP group.", ldapGroupUID, err)
 			errors = append(errors, err)
 			continue
 		}
 		openshiftGroups = append(openshiftGroups, openshiftGroup)
+
+		if !s.DryRun {
+			fmt.Fprintf(s.Out, "group/%s\n", openshiftGroup.Name)
```

Comment:
> Can we do this with the printer instead of in here?

No.  Imagine a very slow LDAP server.  This sync job might take 20 minutes.  In that case, we want to assure the user that something is happening, so they won't cancel it.  This will be ticking off the groups as we go through them.  We use this format so that it can be piped to other commands.